api/src/main/java/net/md_5/bungee/api/connection/Connection.java

```diff
@@ -84,5 +84,20 @@ interface Unsafe
          * @param packet the packet to send
          */
         void sendPacket(DefinedPacket packet);
+
+        /**
+         * Queue a packet to this connection.
+         * If the packet is not registered for the connections current encoder protocol, it will be queued until it is,
+         * otherwise it will be sent immediately.
+         * <p>
+         * If the version of this connection is less than 1.20.2 you should not use this method.
```

Comment:
why not

Reply:
because there is no config state

Reply:
the queued packets method is only for config -> game and game -> config packets
Its unsuported for InitialHandler, we dont need it there

Reply:
At most it'd be a little bit slower due to the extra check, but as functionality is the same, plugins could use both.

Reply:
yes thats right, thats exactly why i have written "you should not"

You can do it but you should not

Reply:
because the old method will the better choice

Reply:
I think I tend to agree with Jan that it should just send immediately on older versions. Consumers of the API (even if unsafe) shouldnt really need to check version unless they have to

Reply:
Checking for >= 1.20.2 now otherwise just fallback to the old sendpacket method

Reply:
and removed the documentation info